share/wake/lib/system/job.wake

```diff
@@ -193,8 +213,10 @@ export def makePlan (label: String) (visible: List Path) (command: String): Plan
   makeShellPlan command visible
   | setPlanLabel label
 
-# Set reasonable defaults for all Plan arguments
+# Identity function
 def id x = x
+
+# Set reasonable defaults for all Plan arguments
```

Comment:
This change was done due to #1207.

As an aside, the `id` function is used both in the `Job.wake` and `Path.wake` files. While out of scope for this particular PR, what do we think about adding an exported identity function in Wake? I think I've created one every now and then myself, and it could be useful. I would probably recommend `def identity` instead of just `def id` though, if we did that, since "id" will be far more commonly used than "identity", as words and function and variable names.